packages/buckets/src/buckets.spec.ts

```diff
@@ -385,24 +390,86 @@ describe('Buckets...', function () {
       const rootKey = root?.key || ''
 
       // Now create a relatively small stream that should finish first
-      const stream2 = new Duplex()
-      stream2.push(Buffer.from('some small amount of text'))
-      stream2.push(Buffer.from('that spans two buffers'))
+      const stream2 = new Readable()
+      stream2.push(Buffer.from('some small amount of text')) // 25 bytes
+      stream2.push(Buffer.from('that spans two buffers')) // 22 bytes
       stream2.push(null)
 
+      let files: File[] = [
+        {
+          path: 'file1.jpg',
+          content: stream1,
+        },
+        {
+          path: 'path/to/file2.txt',
+          content: stream2,
+        },
+      ]
+      const iter = client.pushPaths(rootKey, files, { root })
+      const totals = new Map([
+        ['file1.jpg', 0],
+        ['path/to/file2.txt', 0],
+      ])
+      for await (const { path, root, size } of iter) {
+        expect(['file1.jpg', 'path/to/file2.txt']).to.include(path)
+        expect(root).to.not.be.undefined
+        totals.set(path, size)
+      }
+      // TODO: For some reason I'm getting slightly larger size from buckets
```

Comment:
If the size compared is from the push result, I believe it will be slightly different due to the IPFS adder chunking.

Reply:
Cool, that makes sense. I've updated the test to reflect this.